unit-tests/live/rec-play/test-record-and-stream.py

```diff
@@ -42,40 +42,39 @@ def record(file_name, default_profile):
 
 
 def try_streaming(default_profile):
-    global depth_sensor
+    global depth_sensor, frame_queue
 
     depth_profile = restart_profile(default_profile)
     depth_sensor.open(depth_profile)
-    depth_sensor.start(sync)
+    depth_sensor.start(frame_queue)
     time.sleep(3)
     depth_sensor.stop()
     depth_sensor.close()
 
 
 def play_recording(default_profile):
-    global depth_sensor
+    global depth_sensor, frame_queue
 
     ctx = rs.context()
     playback = ctx.load_device(file_name)
     depth_sensor = playback.first_depth_sensor()
 
     depth_profile = restart_profile(default_profile)
     depth_sensor.open(depth_profile)
-    depth_sensor.start(sync)
+    depth_sensor.start(frame_queue)
```

Comment:
Can we reuse `try_streaming` again maybe here?
and I am not sure but you might want to clear the frame queue between both usage

Reply:
done